src/widgets/_windows.scss

```
@@ -170,10 +170,16 @@ window:not(.popup):not(.menu) {
         background-color: #{"@color_primary"};
 
         .title {
+            font-size: 9pt;
             font-weight: 600;
             text-shadow: $shadow
         }
 
+        .subtitle {
+            opacity: 0.75;
+            font-size: 85%;
```

This is larger than what we had before
```suggestion
            font-size: 80%;
```